nixos/modules/services/backup/tsm.nix

```diff
@@ -0,0 +1,358 @@
+{ config, lib, pkgs, ... }:
+
+let
+
```

Comment:
Please use 
```
inherit (lib) mkOption, strMatching,...;
inherit (builtins) ...;
```
lines, here. Using full-qualified names throughout the code affects readability a lot.

Reply:
Applied, with two exceptions

* I still refer to the sub-namespaces in lib, like `inherit (lib.options) mkOption`. I faintly recall having read somewhere that putting all library elements in the `lib` namespace was a bad idea (I can't find it and I might be wrong about this). While I guess this will never be reverted, I don't want to stand in the way of doing so.
* I didn't add an `inherit` line for libraries with only one element to be imported. In such cases I feel short code weight more than readability (but that's probably a matter of taste).